.env-starter

```diff
@@ -0,0 +1,50 @@
+FB_WEB_API_KEY=                              API key from firebase web app config           e.g. ABC...XYZ                         
```

Comment:
The comments are included in the field values because they are on the same line. Ex. I tried to test the min. period flag - when I deployed the firebase project, the id included "project id from firebase web app config        e.g. project". The comments were also pulled into the client_invite_period field, so I assume it's happening to all of them. We can either put brief comments above each section or I'm good to get rid of them. As you mentioned .env file exchange will be our primary workflow.